robottelo/ui/browser.py

```diff
@@ -109,6 +113,18 @@ def browser():
                 Ie() if settings.webdriver_binary is None
                 else Ie(executable_path=settings.webdriver_binary)
             )
+        elif webdriver_name == 'edge':
+            capabilities = webdriver.DesiredCapabilities.EDGE.copy()
+            capabilities['acceptSslCerts'] = True
```

Comment:
I've tried selenium-3.3.0 (see PR description), still no luck, the issue you mentioned is probably Firefox-specific.
Btw, latest selenium was acting surprisingly good, they probably rolled back the changes which were blocking us as i haven't seen those failures anymore. I think we may give it another try on some test env when we have some time :)